testing/test_pytest_html.py

```diff
@@ -238,9 +238,11 @@ def pytest_runtest_makereport(item, call):
 
     @pytest.mark.parametrize('content, encoded', [
         ("u'\u0081'", 'woE='),
-        ("'foo'", 'Zm9v')])
+        ("'foo'", 'Zm9v'),
+        (b"\xe2\x80\x93" if PY3 else "'\xe2\x80\x93'", "4oCT")])
```

Comment:
Rather than having a condition within a parameter, could we just have an additional parameter with a skip/xfail if necessary for specific versions of Python?